src/style.scss

```diff
@@ -95,17 +96,32 @@ $toolbarHeight: 50px;
     background-size: cover;
 }
 
+.caption,
 .toolbar {
```

Comment:
maybe you prefer those two completely separated? DRY works here, but separating them might also be reasonable.